src/components/Users/UserHome.tsx

```diff
@@ -146,7 +139,7 @@ export default function UserHome(props: UserHomeProps) {
               userData={userData}
               username={username}
               {...props}
-              refetchUserData={refetchUserDetails}
+              refetchUserData={invalidateUserDetails}
```

Comment:
the whole point of being able to invalidate queries is so that, we don't need to pass functions down to props. Let's just call the invalidate query directly wherever we want it.